### PR TITLE
Add AssetStudio 0.15.0

### PR DESCRIPTION
**Pull request by eai04191**
This is my first Manifest, so let me know if you find any mistakes!

It has a checkver, but no autoupdate. because need a JobID and a BuildID to download the artifact from AppVeyor, which requires two http requests.

### CODE REVIEW

**Comment by Calinou** [comment]
Is there a reason you closed this? I was considering merging it.

**Comment by eai04191** [comment]
I figured since I haven't heard back from you on this in a while, you don't want to merge this.
One possible reason is that this is not a game, but a tool for examining game data.
After the few days of waiting for PR to be merged, I thought this might not be appropriate for a game repository.
As a matter of fact, are you going to merge this?
I was going to add other tools that examine game data like this one to this repository if this PR is merged.

If this PR is rejected, I will create my own repository and add it to it. So don't worry about you declining the merge.

<sub>Sorry for the poor English. I'm not good at English!</sub>

**Review by Calinou — APPROVED** [review]
Thanks! Tested locally, it works.

**Pull request merged**
